MagneticField/GeomBuilder/plugins/dd4hep/DD4hep_VolumeBasedMagneticFieldESProducerFromDB.cc

```diff
@@ -0,0 +1,224 @@
+/** \class DD4hep_VolumeBasedMagneticFieldESProducerFromDB
+ *
+ *  Producer for the VolumeBasedMagneticField, taking all inputs (geometry, etc) from DB.
+ *  This version uses DD4hep and is adapted from the DDD version (MagneticField/GeomBuilder/plugins/VolumeBasedMagneticFieldESProducerFromDB.cc)
+ *
+ */
+
+#include "FWCore/Framework/interface/EventSetupRecordIntervalFinder.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+
+#include "CondFormats/RunInfo/interface/RunInfo.h"
+#include "CondFormats/DataRecord/interface/RunSummaryRcd.h"
+
+#include "MagneticField/Engine/interface/MagneticField.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "MagneticField/VolumeBasedEngine/interface/VolumeBasedMagneticField.h"
+#include "MagneticField/ParametrizedEngine/interface/ParametrizedMagneticFieldFactory.h"
+
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/ESProductTag.h"
+
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "MagneticField/GeomBuilder/src/DD4hep_MagGeoBuilder.h"
+
+#include "DetectorDescription/DDCMS/interface/DDCompactView.h"
+#include "DetectorDescription/DDCMS/interface/DDDetector.h"
+
+#include "CondFormats/Common/interface/FileBlob.h"
+#include "CondFormats/DataRecord/interface/MFGeometryFileRcd.h"
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+#include "CondFormats/MFObjects/interface/MagFieldConfig.h"
+#include "CondFormats/DataRecord/interface/MagFieldConfigRcd.h"
+
+#include <string>
+#include <vector>
+#include <iostream>
+#include <memory>
+
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/classification.hpp>
+#include <boost/lexical_cast.hpp>
```

Comment:
I do not see any boost:: in this file; can these includes be dropped?

Reply:
Indeed (then also from plugins/VolumeBasedMagneticFieldESProducerFromDB.cc which, is the original DD4hep version)